lib/electric/migrations/migrations.ex

```diff
@@ -19,7 +19,7 @@ defmodule Electric.Migrations do
   optional argument:
   - :dir where to create the migration rather than the current working directory
   """
-  def init_migrations(options) do
+  def init_migrations(app_id, options) do
```

Comment:
What about the concept of enviornments, or is this a composite id (e.g. `default.lorem-ipsum-245`?)

Reply:
The local migration folder covers all the environments - I thought capturing and storing the app_id here would make the rest of CLI commands neater as its unlikly to change. It's just lorem-ipsum-245 not a composite one